backend/app/endpoints.py

```diff
@@ -20,7 +20,7 @@
 @dataclass
 class PortfolioAsset:
     allocation: float
-    volatility: float
+    metrics: dict[str, float]
```

Comment:
It's a good thing that this PR suggests a change on the backend only, so let's try to make it mergeable by itself, without needing to wait for the frontend changes.

To keep this new backend version compatible with the frontend, we need to keep the `volatility` key here. Let's just add `metrics` next to it.